.gitignore

```diff
@@ -109,3 +109,8 @@ dist
 
 **/target/
 **/.DS_Store
+
+**/bin/
+**/obj/
+
+**/*.feature.cs
```

Comment:
Why is `**/*.feature.cs` added to .gitignore?

Reply:
@ankur-lt on compiling the dotnet package it generates the .cs files. That's why added in the gitignore